src/common/bit_vector.hpp

```diff
@@ -309,7 +309,7 @@ void BitVector<_b>::encode_data(bufferlist& bl, uint64_t byte_offset,
 
   uint64_t end_offset = byte_offset + byte_length;
   while (byte_offset < end_offset) {
-    uint64_t len = MIN(BLOCK_SIZE, end_offset - byte_offset);
+    uint64_t len = std::min((uint64_t)BLOCK_SIZE, end_offset - byte_offset);
```

Comment:
@shinobu-x This PR covers replacement of std::min by MIN.
https://github.com/ceph/ceph/pull/18124
Why we are moving them back?

Reply:
both are moving MIN -> std::min right?